### ISSUE TITLE
習題 13：請給一個我們上課沒講過的機器學習方法程式範例，然後說明其原理（可以用 AI 寫，但你要讀懂）

### ISSUE THREAD

**Issue by ccckmit** [ccc113b/_ml#13]
上課講過的方法如下（不要選這些即可）

* [02-AssemblyAI的實作](https://github.com/ccc113b/py2cs/tree/master/03-%E6%A9%9F%E5%99%A8%E5%AD%B8%E7%BF%92/03-%E6%A9%9F%E5%99%A8%E5%AD%B8%E7%BF%92/01-%E5%82%B3%E7%B5%B1%E6%A9%9F%E5%99%A8%E5%AD%B8%E7%BF%92/02-AssemblyAI%E7%9A%84%E5%AF%A6%E4%BD%9C)

可以使用 sklearn 與 AI (ChatGPT, Claude, Gemini, Grok, ....) ，但是要看懂後解說。

* https://scikit-learn.org/stable/

還有哪些方法

* [List Of All Machine Learning (ML) Algorithms](https://medium.com/@price_kj/list-of-all-machine-learning-ml-algorithms-7c839f8c0d73)

**Comment by wrr606** [comment]
資工三 111110517 范揚玄
https://github.com/wrr606/_ml/blob/main/hw/13/run.py

**Comment by liu99002** [comment]
資工三 111110506 劉銓旭
https://github.com/liu99002/_ml/tree/main/hw/hw13

**Comment by yc0123** [comment]
資工三 111110507 陳穎辰
https://github.com/yc0123/_ml/blob/main/hw13/DBSCAN.py

**Comment by Erkmrcl17** [comment]
資工二 111210559 李麗恩

[HW13](https://github.com/Erkmrcl17/_ml/blob/main/HW13.py)

**Comment by KiuBios** [comment]
資工三 111110519 胡劭家
https://github.com/KiuBios/_ml/tree/main/hw13

**Comment by jm123u** [comment]
資工四 111010577 史煜疏
https://github.com/jm123u/_ml/blob/main/hw13/gradient.py

**Comment by lkhagvaa2444** [comment]
資工三 111110544 李荷葦
https://github.com/lkhagvaa2444/_ml/tree/main/hw13

**Comment by Steventanardi** [comment]
資工三 111110542 林源茂

https://github.com/Steventanardi/_ml/tree/main/Homework13

**Comment by henk3520** [comment]
資工四 111010573 古莛禾
https://github.com/henk3520/_ml/tree/main/hw13

**Comment by Jonathan-es** [comment]
資工二 111210554 陳家盛
https://github.com/Jonathan-es/_ml/blob/main/hw13/hw13.py

**Comment by Yangforrok** [comment]
資工三 111110514 羊柏宇
https://github.com/Yangforrok/_ml/tree/main/13

**Comment by ouhanchen** [comment]
資工三 111110515 歐漢宸
https://github.com/ouhanchen/_ml/tree/main/13

**Comment by Gavinhsu0512** [comment]
資工三 111110501 許元承
https://github.com/Gavinhsu0512/_ml/tree/main/homework13

**Comment by Candy96551** [comment]
資工三 111110502 蔡雨軒
https://github.com/Candy96551/113_ml/blob/main/hw13/Classification.py

**Comment by o5xNo** [comment]
資工三 111110508 吳丞恩
https://github.com/o5xNo/_ml/tree/main/homework/13.ipynb

**Comment by karta9630** [comment]
資工三 111110503 戴嘉豪
https://github.com/karta9630/_ml/blob/main/HW13/1.py

**Comment by peterwang0329** [comment]
資工二 111210513 汪章貴
[https://github.com/peterwang0329/_ml/tree/main/hw13](https://github.com/peterwang0329/_ml/tree/main/hw13)

**Comment by yang930208** [comment]
資工三 111110529 楊智鈞
[作業13](https://github.com/yang930208/_ml/tree/main/%E4%BD%9C%E6%A5%AD13)
[ChatGPT對話連結](https://chatgpt.com/share/6840ff79-b3c4-8008-8123-fc010408188e)

**Comment by sealbrother** [comment]
資工三 111110556 蔡杰叡
https://github.com/sealbrother/_ml/blob/main/hw13.py

**Comment by chen02648** [comment]
資工四 111010539 陳文茜
https://github.com/chen02648/_ml/tree/main/hw13

**Comment by VinsOrl** [comment]
資工二 111210556 林新興
https://github.com/VinsOrl/_ml/blob/main/HW13.py

**Comment by gakuplusq** [comment]
[資工四 黃劭騏 110910530](https://github.com/gakuplusq/_ml/wiki/hw13)

**Comment by XaioYue** [comment]
資工三 111110555 董翔宇
https://github.com/XaioYue/_ml/tree/main/13

**Comment by cindycallista** [comment]
資工二 楊莉思 111210555
https://github.com/cindycallista/_ml/blob/main/HW13.py

**Comment by sam00002135** [comment]
資工四 111010574 白和鑫
https://github.com/sam00002135/ML/blob/main/hm13/hm13.py

**Comment by CJI3751** [comment]
資工四 111010564  潘敏驛
https://github.com/CJI3751/_ml/tree/main/homework/13

**Comment by yakumocirno** [comment]
資工三 111110551 劉品佑
https://github.com/yakumocirno/_ml/tree/main/hw13

**Comment by HuangAlvin** [comment]
資工四 111010526 黃俊皓
https://github.com/HuangAlvin/_ml/tree/main/13

**Comment by ghost** [comment]
資工三 111110525 林思伶
https://github.com/Siling5304/_ml/tree/main/hw13

**Comment by mimmm4308** [comment]
資工四111010525林品翔
https://github.com/mimmm4308/_ml/tree/main/final/13

**Comment by pado35** [comment]
資工三 111110511 李明洋
https://github.com/pado35/_ml/tree/main/13

**Comment by 921006vicky** [comment]
資工三 111110505 劉思妤https://github.com/921006vicky/_ml/tree/main/hw/13

**Comment by Kruez0** [comment]
資工三 111110541 劉佩芬
https://github.com/Kruez0/_ml/tree/main/w13

**Comment by patrick0516** [comment]
資工四 111010575 周辰恩
https://github.com/patrick0516/_ml/tree/main/hw13

**Comment by Logan910115** [comment]
資工四 110910560 羅德耕https://github.com/Logan910115/_ml/tree/main/HW13
